docs/static/releasenotes.asciidoc

```diff
@@ -22,6 +23,39 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-3-3]]
+=== Logstash 8.3.3 Release Notes
+
+[[notable-8.3.3]]
+==== Notable issue fixed
+* We fixed an issue running the plugin manager or the keystore using the bundled JVM on
+  windows https://github.com/elastic/logstash/pull/14355[#14355]
```

Comment:
Please consider adding a bit more info to help users decide if this might have applied to them or not without having to go to the PR. 

Maybe something like: 
> "We fixed an issue which occurred when users ran the plugin manager or the keystore with the bundled JVM. 
Some mandatory JVM options were not being picked up from the JvmOptionsParser, breaking compatibility with Windows on certain versions of the JDK."